docs/requirements.txt

```diff
@@ -1,5 +1,2 @@
-mkdocs<1.1
-jinja2==3.0.0
-mkdocs-awesome-pages-plugin==2.1.0
-markdown<3.4.0
-readthedocs-version-warning-mkdocs-plugin
+mkdocs==1.6.1
```

Comment:
I would like to recommend you look at this plugin "framework" https://github.com/squidfunk/mkdocs-material

It's a powerful tool that can give us more options while making the whole documentation more transparent.

Reply:
I have checked it, but I probably don't know what you have in mind. We may talk about it sometime